src/units/abilities.cpp

```diff
@@ -1135,6 +1209,19 @@ void attack_type::weapon_specials_impl_adj(
 				add_name(temp_string, active, cfg["name"].str(), checking_name);
 			}
 		}
+
+		for(const unit& unit_itor : units){
+			if (!unit_itor.has_ability_distant() || unit_itor.incapacitated() || &(unit_itor) == self.get()) {
+				continue;
+			}
+			for(const auto [key, cfg] : unit_itor.abilities().all_children_view()) {
+				bool tag_checked = (!checking_tags.empty()) ? (checking_tags.count(key) != 0) : true;
+				bool default_bool = (affect_adjacents == "affect_allies") ? true : false;
+				bool affect_allies = (!affect_adjacents.empty()) ? cfg[affect_adjacents].to_bool(default_bool) : true;
+				const bool active = tag_checked && check_dist_abilities_impl(self_attack, other_attack, cfg, self, unit_itor, self_loc, unit_itor.get_location(), whom, key, leader_bool) && affect_allies;
+				add_name(temp_string, active, cfg["name"].str(), checking_name);
+			}
```

Comment:
lost of extra parenthesis in this for loop

Reply:
fixed